docs/overview.rst

```diff
@@ -177,27 +177,27 @@ Key features
-| <Kinto/kinto-http.js/>`                     | <Kinto/kinto.py>`                           |
-+---------------------------------------------+---------------------------------------------+
-| |logo-offline|                              | |logo-admin|                                |
-| Offline-first `JavaScript client            | :github:`Web Admin UI                       |
-| <https://kintojs.readthedocs.io>`_          | <Kinto/kinto-admin>`                        |
-+---------------------------------------------+---------------------------------------------+
-| |logo-history|                              | |logo-livesync|                             |
-| :ref:`History of changes and authorship     | Live :ref:`Push notifications               |
-| <api-history>`                              | <tutorials>`                                |
-+---------------------------------------------+---------------------------------------------+
-| |logo-attachment|                           | |logo-signature|                            |
-| :github:`File attachments on records        | :github:`Digital signature and review       |
-| <Kinto/kinto-attachment>`                   | workflows <Kinto/kinto-signer>`             |
-+---------------------------------------------+---------------------------------------------+
-| |logo-quotas|                               | |logo-boilerplate|                          |
-| :ref:`Storage quotas                        | :github:`Kinto+React boilerplate            |
-| <api-quotas>`                               | <Kinto/kinto-react-boilerplate>`            |
-+---------------------------------------------+---------------------------------------------+
++---------------------------------------------+---------------------------------------------------------+
+| |logo-javascript|                           | |logo-python|                                           |
+| :github:`JavaScript HTTP API client         | :github:`Python HTTP API client                         |
+| <Kinto/kinto-http.js/>`                     | <Kinto/kinto.py>`                                       |
++---------------------------------------------+---------------------------------------------------------+
+| |logo-offline|                              | |logo-admin|                                            |
+| Offline-first `JavaScript client            | :github:`Web Admin UI                                   |
+| <https://kintojs.readthedocs.io>`_          | <Kinto/kinto-admin>`                                    |
++---------------------------------------------+---------------------------------------------------------+
+| |logo-history|                              | |logo-livesync|                                         |
+| :ref:`History of changes and authorship     | Live `Push notifications                                |
+| <api-history>`                              | <https://github.com/webpush-channels/webpush-channels>`_|
```

Comment:
Here I would rather put a link to a Kinto plugin or documentation page. The webpush-channels project seems to be an independant project, no? (yes, using kinto.core but how does it integrate with kinto in the end?)